### ISSUE TITLE
Object.extname is deprecated.

### ISSUE THREAD

**Issue by joaosaro** [ManRueda/jade-autocompile#23]
Argument to `path.extname` must be a string
```
Object.extname (/Applications/Atom.app/Contents/Resources/app.asar/src/electron-shims.js:20:10)
Object.toggle (/Users/joaosaro/.atom/packages/jade-autocompile/lib/jade-autocompile.js:208:28)
<unknown> (/Users/joaosaro/.atom/packages/jade-autocompile/lib/jade-autocompile.js:75:15)
Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/pane-container.js:348:27)
```

**Comment by Fededark** [comment]
![immagine](https://cloud.githubusercontent.com/assets/6904824/22295866/ae273840-e318-11e6-9a4f-90270866d4d5.png)

**Comment by techiedodo** [comment]
~~Had the same warning. I updated all packages on Atom and the warning went away. Have you guys tried updating all packages on Atom? Could it be an issue with another packaged installed?~~
---UPDATE---
Disregard previous comment. Error came back up when atom was reopened once again.

**Comment by uxdiva** [comment]
I'm getting the same deprecation warning.

**Comment by manrueda** [comment]
Hi guys, thanks you for the warning. I'm a little bit busy right now, but i think that i will resolve it for the end of this week.

**Comment by manrueda** [comment]
Should be fixed in the version 0.8.5
